packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -109,10 +106,14 @@ class ClayMultiSelect extends ClayComponent {
 
 		if (char === ',' || words.length > 1) {
 			words.forEach(word => {
-				this._addLabelItem(this._createItemObject(word, word));
-			});
+				let added = this._addLabelItem(
+					this._createItemObject(word, word)
+				);
 
-			inputValue = '';
+				if (added) {
```

Comment:
It seems to have some pretty strange cases when I type 'qwe' and adds the item, then I type again and add a comma and type a new one after the comma it will add but the values ​​will still remain there:

<img src="http://g.recordit.co/ap401jRtdh.gif" />

Reply:
We must make sure to delete the commas to avoid problems, like adding an existing value, adding the comma and then pressing enter.

Reply:
Do you think we can iterate over this? Or must be fixed on this pr?

Reply:
I think it's a bit critical to iterate, I think it's best to fix this in PR.